parachain/runtime/interlay/src/lib.rs

```diff
@@ -168,27 +168,22 @@ impl Contains<RuntimeCall> for BaseCallFilter {
         if matches!(
             call,
             RuntimeCall::System(_)
-                | RuntimeCall::Authorship(_)
-                | RuntimeCall::Session(_)
                 | RuntimeCall::Timestamp(_)
-                | RuntimeCall::ParachainSystem(_)
```

Comment:
Why remove these? I'm not sure what would happen if these were paused, have you tested that?

Reply:
Good catch, it's a copy-paste mistake, I will fix. I copied from standalone, which doesn't have these pallets. There is an integration test for all pallets that are in the standalone runtime, but obviously that can't test these there. We'll be able to test these properly once we get rid of standalone but for now I think it's fine - just don't pause these pallets :)

Reply:
@sander2 on a related note I noticed the other runtimes don't catch `RuntimeCall::Session(_)` in the filter, up to you if we add that there as well